cmd/dependabot/internal/cmd/graph.go

```diff
@@ -7,18 +7,12 @@ import (
 	"io"
 	"log"
 	"os"
-	"slices"
 
 	"github.com/MakeNowJust/heredoc"
 	"github.com/dependabot/cli/internal/infra"
 	"github.com/spf13/cobra"
 )
 
-var graphSupportedEcosystems = []string{
-	"bundler",
-	"go_modules",
-}
-
```

Comment:
👍🏻 - this no longer makes sense to restrict as you say, I'm in queue to merge the new version of the experiment which means most ecosystems will now have a base-line capability vs the previous approach where we had to do enabling work.